emerlin2caom2/main_app.py

```diff
@@ -35,9 +35,37 @@ def emcp2dict(emcp_file):
         if ':' in line_no_space:
             nested_list = line_no_space.split(':')
             pickle_dict[nested_list[0]] = nested_list[1]
+    pickle_dict['target_position'] = target_position(emcp_file, pickle_dict['targets'])
     return pickle_dict
 
 
+def target_position(emcp_file, target):
```

Comment:
I thought the plan was just to get primary target name and then select the target position from the MS based on that name using the measurement set Field table (that msmd function should do this for phase centre or reference), so that whatever 'final' observed position is included with the data product, rather than whatever position the PI/proposer submitted.  This absolutely fine for now though.